examples/cosign/cosign.cue

```diff
@@ -0,0 +1,37 @@
+package ssf
+
+_image: {
+	name: "slsa"
+}
+
+
+pipelineRun: "ko-pipelinerun-": spec: {
+	pipelineRef: name: "ko"
+	params: [ {
+		name:  "APP_IMAGE"
+		value: _APP_IMAGE
+	}, {
+		name:  "imageUrl"
+		value: _APP_IMAGE
+	}, {
+		name:  "imageTag"
+		value: "latest"
+	}, {
```

Comment:
These parameters don't seem to be used in the pipeline definition.

Reply:
Addressed ✅